lib/pengine/common.c

```diff
@@ -206,28 +206,28 @@ static pcmk__cluster_option_t pe_opts[] = {
         "false", pcmk__valid_boolean,
         N_("*** Deprecated *** Whether to remove stopped resources from "
             "the executor"),
-        "Values other than default are poorly tested and potentially dangerous."
-            " This option will be removed in a future release."
+        N_("Values other than default are poorly tested and potentially dangerous."
+            " This option will be removed in a future release.")
```

Comment:
This is being covered by:

https://github.com/ClusterLabs/pacemaker/pull/2988/files#diff-ea6b533265f0bb598c3d6045482e7dbb15f12ea70e3074b7a75f72dfff5be199R209

Let's probably go with that and wait for it to be merged.

Reply:
Apologies for the delay. That PR has been merged now, so this one can be rebased on the current main branch to resolve the conflict.